docs/data/charts/styling/CustomOverlay.js

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Stack from '@mui/material/Stack';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { useDrawingArea, useXScale, useYScale } from '@mui/x-charts/hooks';
+
+const ratios = [0.2, 0.8, 0.6, 0.5];
+
+const LoadingReact = styled('rect')({
+  opacity: 0.2,
+  fill: 'lightgray',
+});
+
+const LoadingText = styled('text')(({ theme }) => ({
+  stroke: 'none',
+  fill: theme.palette.text.primary,
+  shapeRendering: 'crispEdges',
```

Comment:
We are missing the font here no? I would expect to see theme.typography.body1 or body2 depending on what makes the most sense (maybe body2?)